app/forms/FlatRateForms.scala

```diff
@@ -98,11 +98,13 @@ object EstimateTotalSalesForm {
   implicit val errorCode: ErrorCode = "frs.estimateTotalSales"
   val regex = """^[0-9 ,]*\.?[0-9]+$""".r
```

Comment:
can we drop the inclusion of comma in this and regex on line 101? i know it wouldn't make any difference as the next one disallows it with an error. same for other forms

Reply:
That was intentional, without it the comma specific error message wasn't being displayed as it used resolve it to generic error of invalid format when comma is present. However by changing the order of the regexes this can be avoided. So I am going to remove the comma from the regex suggested by you.